pkgs/applications/networking/browsers/yandex-browser/default.nix

```diff
@@ -132,5 +132,12 @@ stdenv.mkDerivation rec {
     license = licenses.unfree;
     maintainers = with maintainers; [ dan4ik605743 ];
     platforms = [ "x86_64-linux" ];
+
+    knownVulnerabilities = [
+      ''
+      Trusts a Russian government issued CA certificate for some websites.
+      See https://habr.com/en/company/yandex/blog/655185/ (Russian) for details.
```

Comment:
Would it be worth it or ideal to archive this page in its current form and link to that (e.g. using some service like archive.org's Wayback Machine)?

Reply:
http://web.archive.org/web/20220311160825/https://habr.com/en/company/yandex/blog/655185/